src/cli/index.js

```diff
@@ -362,7 +362,7 @@ config.init({
   binLinks: commander.binLinks,
   modulesFolder: commander.modulesFolder,
   globalFolder: commander.globalFolder,
-  cacheRootFolder: commander.cacheFolder,
+  cacheFolder: commander.cacheFolder,
```

Comment:
This probably does need a change to either `_cacheRootFolder` or `cacheFolder` since `cacheRootFolder` does not exist in the configOptions export type. The reason for this change was the cache folder location not changing when setting it with the `--cache-folder` command (see #2881).